src/types.jl

```diff
@@ -136,6 +136,7 @@ function arguments(x::BasicSymbolic)
     end
     return args
 end
+children(x::BasicSymbolic) = arguments(x)
```

Comment:
this could be fishy

Reply:
this was what I was thinking as well, but then stumbled over `arguments` calling `unsorted_arguments` which is part of TI, so I kept it the current way. should we rename `unsorted_arguments` as well then? 🤔

Reply:
Ok, sounds good for now. Maybe we should file an issue to TI directly. cc @shashi

Reply:
Shouldnt children also include the operation at the beginning?